app/helpers/users_helper.rb

```diff
@@ -24,4 +24,18 @@ def user_taxonomies_html_options(user)
       :organization => { :onchange => 'taxonomy_added(this, "organization")'}
     } unless user.admin?
   end
+
+  def mail_notification_settings(mail_notification)
+    (_(mail_notification.description) + mail_notification_query_builder_settings(mail_notification)).html_safe
+  end
+
+  def mail_notification_query_builder_settings(mail_notification)
+    mail_notification.queryable? ? link_to_function(icon_text('cog', "", { :class => "#{mail_notification.name}_query_setting",
```

Comment:
The cog seems rather hidden to me, could we perhaps always show the field and associate it with the dropdown menu for the notification, e.g. left indentation or surrounding box